src/pages/dmt5/_interest_free_trading.js

```diff
@@ -171,7 +172,12 @@ const InterestFreeTrading = () => {
                 <StyledRightText>
                     <Localize translate_text="Enjoy interest-free trading with no swap charges on overnight positions for forex, stocks, stock indices, commodities, cryptocurrencies, and synthetic indices." />
                 </StyledRightText>
-                <StyledLinkButton secondary="true" to="" external="true" target="_blank">
+                <StyledLinkButton
+                    secondary="true"
+                    to={deriv_mt5_app_url}
```

Comment:
```suggestion
                    type="mt5"
```

Reply:
This should be set via the `type` instead. You can refer to `localized-link.js` for more info on the current config.